.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
 env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+  - DISPLAY=:99.0
```

Comment:
Ok I didn't know it was possible to give such details in the matrix. I will try if something could be done.